src/common/RootProviders.tsx

```diff
@@ -22,6 +23,9 @@ export function RootProviders({
   authConfig: AuthConfiguration;
   children?: React.ReactNode;
 }) {
+  const { theme: customTheme } = useDeveloperConfig();
+  const theme = customTheme ?? {};
```

Comment:
The `theme` provided to `BrandConfigProvider` can be undefined so this isn't necessary.